amazon-connect/callstats-amazon-connect-shim.js

```diff
@@ -11,6 +11,11 @@
       this.fftBins = new Float32Array(this.analyser.frequencyBinCount);
       this.mediaStreamSource = this.audioContext.createMediaStreamSource(stream);
       this.mediaStreamSource.connect(this.analyser);
+
+      this.audioProcessor = this.audioContext.createScriptProcessor(512);
```

Comment:
audioProcessor as well as the analyzer have a window of 512 samples. how often are corresponding functions called?
with a sampling rate of 44kHz there are 44000 samples per second, so 512 samples would only show the last 11.6 msecs